cmake/configs/aarch64/linux/config.cmake

```diff
@@ -12,9 +12,15 @@ set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -D_GNU_SOURCE" CACHE STRING "" FORCE)
 # Linux requires buffers aligned to 4KB boundaries for O_DIRECT to work.
 set(WT_BUFFER_ALIGNMENT_DEFAULT "4096" CACHE STRING "")
 
-# ARMv8-A is the 64-bit ARM architecture, turn on the optional CRC instructions.
-set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -march=armv8-a+crc" CACHE STRING "" FORCE)
-
+# ARMv8-A is the 64-bit ARM architecture, turn on the optional CRC.
+if (HAVE_RCPC)
+    # If the builder specifies also turn on the RCpc instructions.
+    set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -march=armv8.2-a+rcpc+crc" CACHE STRING "" FORCE)
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -march=armv8.2-a+rcpc+crc" CACHE STRING "" FORCE)
```

Comment:
Are C++ flags now required, or are we adding them for consistency?

Reply:
Need them for unit testing, i'll push up the unit test.